src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Easy OTP on the web with the SMS Receiver API
```

Comment:
```suggestion
title: Easy one-time passwords (OTP) on the web with the SMS Receiver API
```

This may help SEO a little.

Reply:
IMO people knows about OTP but "one-time passwords". I would leave it as is.